src/bitcoin-tx.cpp

```diff
@@ -412,6 +428,12 @@ static void MutateTxSign(CMutableTransaction& tx, const string& flagStr)
 
     bool fHashSingle = ((nHashType & ~SIGHASH_ANYONECANPAY) == SIGHASH_SINGLE);
 
+    // TODO: Need to pass this in
+    int nHeight = 0;
```

Comment:
This height needs to be updated as comment says, otherwise this method always creates a pre-overwintered signature, I think. Maybe instead of passing height you can check if the overwintered flag is set in `tx`.

Reply:
This comment doesn't affect the type of signature; the code already uses `tx.fOverwintered` to decide what type of signature to produce. What the height here does is pick which branch ID the signature commits to, which *does* need to be passed in, because it requires context that this standalone binary does not have.

Reply:
OK, so you will add a commit/change this commit to pass it in in this PR?